source/dub/generators/build.d

```diff
@@ -524,7 +530,7 @@ class BuildGenerator : ProjectGenerator {
 				if (!exe_path_string.startsWith(".") && (exe_path_string.length < 2 || exe_path_string[1] != ':'))
 					exe_path_string = ".\\" ~ exe_path_string;
 			}
-			logInfo("Running %s %s", exe_path_string, run_args.join(" "));
+			logInfo("Running", Color.green, "%s %s", exe_path_string, run_args.join(" "));
```

Comment:
can you make sure before running the console gets reset?

Reply:
Uhm, that shouldn't be necessary, does it? The ASCII escape sequences are all "terminated" after being used. I mean I can't see a way for that not being the case since colored text is always generated with the `color()` method that always puts an escape sequence at the beginning and one at the end that restores the state (on Posix). On Windows, the `cwrite()` method that is used to print colored text, as soon as the `WinTermEmulation` struct gets destroyed, resets everything as it was before.

Or did you mean something else that I'm not aware of?

Reply:
hm sure just wanted to make sure this was thought about